ush/config.sh.RRFS_NA_13km

```diff
@@ -0,0 +1,87 @@
+MACHINE="jet"
+ACCOUNT="nrtrr"
+RESERVATION="rrfsdet"
+EXPT_BASEDIR="/home/rtrr/RRFS"
+EXPT_SUBDIR="RRFS_NA_13km"
+
+if [[ -n $RESERVATION ]] ; then
+  ACCOUNT=rtrr
+  PARTITION_DEFAULT=kjet
+  PARTITION_FCST=kjet
+  PARTITION_GRAPHICS=kjet
+fi
+
+FIXLAM_NCO_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
+FIXgsm=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_am.20210128
+GWD_HRRRsuite_BASEDIR=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_lam.20210128
+
+VERBOSE="TRUE"
+
+RUN_ENVIR="nco"
+PREEXISTING_DIR_METHOD="rename"
+
+PREDEF_GRID_NAME=GSD_RAP13km
+
+QUILTING="TRUE"
+CCPP_PHYS_SUITE="FV3_GSD_v0"
+FCST_LEN_HRS="12"
+LBC_SPEC_INTVL_HRS="6"
+
+HALO_BLEND=10
+
+WTIME_RUN_FCST="02:00:00"
+
+DATE_FIRST_CYCL="20210328"
+DATE_LAST_CYCL="20220328"
+CYCL_HRS=( "00" "06" "12" "18")
+
+EXTRN_MDL_NAME_ICS="FV3GFS"
+EXTRN_MDL_NAME_LBCS="FV3GFS"
+EXTRN_MDL_LBCS_FILE_NUM="14"  # 14th file is a 39-h forecast from GFS
+
+FV3GFS_FILE_FMT_ICS="grib2"
+FV3GFS_FILE_FMT_LBCS="grib2"
+
+envir="para"
+
+NET="RRFS_NA_13km"
+TAG="RRFS_NA_13km"
```

Comment:
TAG should include "dev1", so maybe "RRFS_dev1_NA_13km" to follow what's happening with CONUS and AK now.

Reply:
Will do—what about "NET"?  I'm not sure what it does.

Reply:
It looks like NET sets the prefix for the grib files out of post, and is included as part of the PTMP directory structure under `com`.